### PR TITLE
Added a changelog file.

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
@kshakir @mcovarr

**Comment by kshakir** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Cromwell Change Log
+
+0.18:
+=====
+* Workflow options can now include a "defaultRuntimeAttributes" section, so that the same runtime attribute is not needed in every single WDL task.
+* Changed the JES parameters named defaultDisks and defaultZones to be simply disks and zones respectively.
```
Comment (mcovarr) [inline CHANGELOG.md]: maybe some backticks around the attribute names

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**